src/routes/+page.svelte

```diff
@@ -22,13 +23,18 @@
 
 <div>
     <h1 class="text-3xl font-bold underline">Hello world!</h1>
+    <TextField
+        textFieldText="Enter Recipient Address or EOS Account"
+        validator={(val) => /^0x[a-fA-F0-9]{40}$/.test(val) || /^[a-zA-Z0-9]+$/.test(val)}
+        errorText="Incorrect wallet format."
+    />
     <div class="m-2">
         <PrimaryButton primaryButtonInputs={buttonInputs} />
     </div>
 </div>
 
 <style lang="postcss">
     :global(html) {
-        background-color: theme(colors.green.800);
+        background-color: #1e1e1e;
```

Comment:
this has been updated I'm guessing you may need to merge main to get all the previous updates
```<style lang="postcss">
    :global(html) {
        background-color: theme(colors.black);
    }
</style>```